internal/server/v1/propertyvalues/linked.go

```diff
@@ -85,3 +85,73 @@ func LinkedPropertyValues(
 	}
 	return result, nil
 }
+
+// BulkLinkedPropertyValues implements mixer.BulkLinkedPropertyValues handler.
+func BulkLinkedPropertyValues(
+	ctx context.Context,
+	in *pb.BulkLinkedPropertyValuesRequest,
+	store *store.Store,
+) (*pb.BulkPropertyValuesResponse, error) {
+	property := in.GetProperty()
+	entities := in.GetEntities()
+	valueEntityType := in.GetValueEntityType()
+	// Check arguments
+	if property != "containedInPlace" {
+		return nil, status.Errorf(
+			codes.InvalidArgument, "only support property 'containedInPlace'")
+	}
+	if valueEntityType == "" {
+		return nil, status.Errorf(
+			codes.InvalidArgument, "missing argument: value_entity_type")
+	}
+	if !util.CheckValidDCIDs(entities) {
+		return nil, status.Errorf(
+			codes.InvalidArgument, "invalid entities %s", entities)
+	}
+	resp, err := placein.GetPlacesIn(
+		ctx,
+		store,
+		entities,
+		valueEntityType,
+	)
+	if err != nil {
+		return nil, err
+	}
+	valueDcids := []string{}
+	for _, e := range resp {
+		valueDcids = append(valueDcids, e...)
+	}
+	// Fetch names
+	data, _, err := Fetch(
+		ctx,
+		store,
+		[]string{"name"},
+		valueDcids,
+		0,
+		"",
+		"out",
+	)
+	if err != nil {
+		return nil, err
+	}
+	result := &pb.BulkPropertyValuesResponse{}
+	for e, children := range resp {
+		tmp := &pb.BulkPropertyValuesResponse_EntityPropertyValues{
+			Entity: e,
+		}
+		for _, dcid := range children {
+			var name string
+			if tmp, ok := data["name"][dcid]; ok {
```

Comment:
There is already a "tmp" outside the for-loop scope, so use a different name here? Otherwise it'll be a bit confusing to read.

Also, instead of "tmp", could we have a more meaningful name?

Reply:
Done. Use meaningful names in both locations.